### PR TITLE
Adding The Gunslinger class

### PR DESCRIPTION
**Pull request by ForbiddenZex**
This is a brew for the Heavyarms Gunslinger, I just finished it and would like it to be added to the repo. Please let me know if I need to make any changes.

### CODE REVIEW

**Review by TheGiddyLimit — APPROVED** [review]

**Comment by ForbiddenZex** [comment]
What is exit code 1?

File: class/Heavyarms; Gunslinger.json [hunk 1]
```diff
@@ -0,0 +1,2882 @@
...
+			"tables": [
+				{
+					"name": "",
+					"source": "",
+					"type": "table",
+					"intro": [
+						"The following table presents the average damage of the gunslinger based on a Monte Carlo method simulating 100,000 rounds of attacks at each level against a Normal Distribution of Armor Class values appropriate for that level. Similar data sets using existing builds are also presented following the same methodology for comparative purposes."
+					],
+					"colLabels": [
+						"",
```
Comment (TheGiddyLimit) [inline class/Heavyarms; Gunslinger.json:1689]: should be strings, then use `text-center` in `colStyles`

**Review by TheGiddyLimit — CHANGES_REQUESTED** [review]
missclicked that "approve" button tbh!

**Comment by TheGiddyLimit** [comment]
> What is exit code 1?

Standard computer nonsense for "it failed." Scroll up to find the logs: https://github.com/TheGiddyLimit/homebrew/pull/1385/checks#step:5:969

File: class/Heavyarms; Gunslinger.json (hunk 1 above)
Comment (ForbiddenZex) [reply class/Heavyarms; Gunslinger.json:1689]: Ah so the exit code is default for whatever

Comment (ForbiddenZex) [reply class/Heavyarms; Gunslinger.json:1689]: I remember this part, Lyra actually wrote it. It functions fine but I remember him saying you might not approve it lol

Comment (TheGiddyLimit) [reply class/Heavyarms; Gunslinger.json:1689]: and he was correct
any reason my proposed fix doesn't work?

Comment (ForbiddenZex) [reply class/Heavyarms; Gunslinger.json:1689]: finished the changes

**Pull request merged**